mkdocs.yml

```diff
@@ -67,23 +30,21 @@ nav:
           - components/qa-service/index.md
           - Semantic Search: components/qa-service/semantic-search.md
           - LLM Response: components/qa-service/llm-response.md
-      - Triaging Service:
-          - components/triaging/index.md
-  - Other Components:
-      - other-components/index.md
       - LLM Proxy Server:
-          - other-components/litellm-proxy/index.md
+          - components/litellm-proxy/index.md
       - Align Score:
-          - other-components/align-score/index.md
-          - Deployment: other-components/align-score/deployment.md
-          - Testing: other-components/align-score/testing.md
+          - components/align-score/index.md
+          - Deployment: components/align-score/deployment.md
+          - Testing: components/align-score/testing.md
       - Chat Managers:
-          - other-components/chat_managers/index.md
-          - Typebot: other-components/chat_managers/typebot.md
-          - Botpress: other-components/chat_managers/botpress_v12.md
+          - components/chat_managers/index.md
+          - Typebot: components/chat_managers/typebot.md
+          - Botpress: components/chat_managers/botpress_v12.md
       - WhatsApp Connector:
-          - Setup: other-components/whatsapp/setup.md
-          - Testing: other-components/whatsapp/testing.md
+          - components/whatsapp/index.md
+          - Setup: components/whatsapp/setup.md
+          - Testing: components/whatsapp/testing.md
+
   - Deploying:
```

Comment:
Overall, I'm thinking about what AAQ users will want to see.. and this is a nit-picky one but perhaps we can call this "Deploy" instead of Deploying?

Reply:
agreed